examples/postgres-RDS/README.md

```diff
@@ -0,0 +1,296 @@
+# Postgres-RDS
+
+[PostgreSQL](https://www.postgresql.org/) is an object-relational database management system (ORDBMS) with an emphasis on extensibility and standards compliance.
+
+## Introduction
+
+This example is to demonstrate how Kanister can be used to import data in AWS RDS instance from On-prem Postgres using Kanister functions
+
+## Prerequisites
+
+- Kubernetes 1.10+
+- PV provisioner support in the underlying infrastructure
+- Kanister controller version 0.84.0 installed in your cluster
+- Kanctl CLI installed (https://docs.kanister.io/tooling.html#kanctl)
+
+## Create RDS instance on AWS
+
+We need to have a Postgres RDS instance where we would restore the data.
+
+> You can skip this step if you already have an RDS instance created
+
+RDS instance needs to be reachable from the outside world. So make sure that you have VPC with the security group having the rule to allow ingress traffic on 5432 TCP port.
+
+
+You can create a security group and add rules to the default VPC using the following commands
+
+```bash
+aws ec2 create-security-group --group-name pgtest-sg --description "pgtest security group"
+aws ec2 authorize-security-group-ingress --group-name pgtest-sg --protocol tcp --port 5432 --cidr 0.0.0.0/0
```

Comment:
Can we add a note to say this is highly insecure and shouldn't be use in a production environment/is only good for testing?
That rule basically makes this database accessible from any source IP in the entire internet ;)